packages/http-server-csharp/test/diagnostic.test.ts

```diff
@@ -1,14 +1,16 @@
-import { BasicTestRunner, expectDiagnostics } from "@typespec/compiler/testing";
+import { expectDiagnostics, TesterInstance } from "@typespec/compiler/testing";
 import { beforeEach, it } from "vitest";
-import { createCSharpServiceEmitterTestRunner, getStandardService } from "./test-host.js";
-let runner: BasicTestRunner;
+import { compileAndDiagnose, getStandardService, Tester } from "./test-host.js";
+
+let tester: TesterInstance;
 
 beforeEach(async () => {
-  runner = await createCSharpServiceEmitterTestRunner();
+  tester = await Tester.createInstance();
 });
 
 it("warns for custom scalars", async () => {
-  const [_, diagnostics] = await runner.compileAndDiagnose(
+  const [_, diagnostics] = await compileAndDiagnose(
```

Comment:
ideally those should get converted to either
```ts
const diagnostics = await tester.diagnose(
```
or if the tester is not used for anything above could just cut the beforeEach and do 
```ts
const diagnostics = await Tester.diagnose(
```